services/movie.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list = None,
+        actors_ids: list = None
+) -> models.QuerySet:
+
+    movies = Movie.objects.all()
+    if genres_ids:
+        movies = movies.filter(genre__id__in=genres_ids)
+    if actors_ids:
+        movies = movies.filter(actor__id__in=actors_ids)

```

Comment:
The filter should use `actors__id__in` instead of `actor__id__in` to match the `ManyToManyField` name `actors` in the `Movie` model.